motoman_ms210_moveit_config/launch/sensor_manager.launch.xml

```diff
@@ -0,0 +1,17 @@
+<launch>
+
+  <!-- This file makes it easy to include the settings for sensor managers -->
+
+  <!-- Params for 3D sensors config -->
+  <rosparam command="load" file="$(find motoman_ms210_moveit_config)/config/sensors_3d.yaml" />
```

Comment:
Introduces error note: `Failed to find 3D sensor plugin parameters for octomap generation`
I think this is okay as it will highlight to users interested in adding a 3D Sensor that this line will need uncommented and sensors added to the list in `sensors_3d.yaml`